test-app/src/widgets/AppWindow.js

```diff
@@ -809,22 +909,28 @@ export default class AppWindow extends createjs.Container {
       mailingList._unhighlightAll();
       mailingList.accessible.selected = [];
       combobox.text = '';
+      emailField._updateDisplayString('');
+      emailField.onBlur();
+      addressField._updateDisplayString('');
+      addressField.onBlur();
     };
 
     const submitBtnData = {
-      type: 'button',
+      type: 'submit',
       value: 'SUBMIT',
       name: 'SUBMIT',
       enabled: true,
       autoFocus: false,
-      form: 'form_1',
+      // form: 'form_1',
```

Comment:
Just a question cause I'm trying to understand how the button and form work, why is this commented out?

Reply:
I did comment that out initially because I thought it might be creating an issue with the form submission since the `form` attribute takes the form id and form id isn't `form_1`. Form id is actually `acc_167`.